README.md

```diff
@@ -135,6 +135,23 @@ sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more infor
 
 Similar to configuring the server, the client configuration file is located at `/etc/opt/omi/conf/omicli.conf`.
 
+### Collecting Performance Metrics using OMI/OMI Provider on docker container or virtual machine
+
+OMI/OMI provider can work on containers and machines, just make sure you know following notes:
+- OMI/OMI provider can collect virtual machine performance metrics even in a normal container.
+  - To do this for container, there are 3 ways docker supports this: pseudo-files, docker stats command and the Remote API, and Docker doesn't need to run with a privileged container.
+  - To do this for virtual machine, you don't need to use any docker APIs and just collect perf metrics as in physical machine.
+- Multiple OMI instances can run in different containers if they are mapped to different ports on the host system. These instances can simultaneously collect VM-based perf metrics without any issue. Additionally, any one OMI instance can run multiple OMI providers without issue.
+- OMI itself does not ship with a provider that collects container performance metrics. OMI does not ship with any providers at all, these must be installed separately.
+  - To be able to collect container performance metrics, you can use [ContainerStatisticsProvider], or you can implement a similar provider. If you wish to use [ContainerStatisticsProvider], this ships with [OMS]. You can either install [OMS] itself, or you can extract [ContainerStatisticsProvider] from the [OMS] bundle and install that provider independently.
```

Comment:
Please combine your statement from line 142 with line 146. 
To be able to collect container performance metrics, there are three ways docker supports this: pseudo-files, docker stats command and the Remote API, and Docker doesn't need to run with a privileged container. you can use [ContainerStatisticsProvider], or you can implement a similar provider. If you wish to use [ContainerStatisticsProvider], this ships with [OMS]. You can either install [OMS] itself, or you can extract [ContainerStatisticsProvider] from the [OMS] bundle and install that provider independently.